honestcash-v2/src/app/modules/editor/embed/embed.component.html

```diff
@@ -0,0 +1,153 @@
+<ng-template #paywallPopContent>For more information about pay-to-read sections, please refer to <a href="https://honest.cash/honest_cash/earn-more-money-for-your-content-paywalls-3552" target="_blank">Paywalls Guide</a></ng-template>
+<ng-template #paywallPopTitle>About Pay-to-read Sections</ng-template>
+<div #modalBody class="modal-body">
+  <div class="panes" [@slide]="activePane">
+    <div class="pane"><div id="editor"></div></div>
+    <div class="pane ml-4 p-4">
+      <form [hidden]="activePane === 'first'" name="form" class="needs-validation" #f="ngForm" (ngSubmit)="f.form.valid && onSubmit(f)" novalidate>
+        <div class="form-row">
+          <div class="col">
+            <div class="form-group">
+              <label for="title">Title</label>
+              <input
+                required
+                minlength="1"
+                id="title"
+                name="title"
+                #title="ngModel"
+                [ngModel]="story.title"
+                [ngClass]="{ 'is-invalid': f.submitted && title.invalid, 'is-valid': f.submitted && title.valid }"
+                class="form-control"
+                type="text"
+                placeholder="Title of your story"
+              >
+              <div
+                class="invalid-feedback"
+                *ngIf="f.submitted && title.invalid"
+              >
+        <span
+          *ngIf="title.errors.required"
+        >
+          Title is required.
+        </span>
+                <span
+                  *ngIf="title.errors.minlength"
+                >
+          Title must be at least 1 character long.
```

Comment:
Change to "title is required".

Reply:
Case 1:
User has nothing written. he sees Title is required
Case 2:
User has one character written. he sees Title is required
Case 3:
User has two characters written. he sees nothing

So let's debate whether a title with one character or two characters are real titles and whether we should explicitly show that it must be atleast 1 character long or mislead?